.travis.yml

```diff
@@ -43,38 +53,33 @@ matrix:
   - os: linux
     compiler: arm-linux-gnueabihf-gcc
     env: OOT=0 TEST=FAST SDE=0 THR="none" CONF="cortexa15" \
-      PACKAGES="gcc-arm-linux-gnueabihf qemu-system-arm qemu-user" \
+      PACKAGES="gcc-arm-linux-gnueabihf libc6-dev-armhf-cross qemu-system-arm qemu-user" \
       TESTSUITE_WRAPPER="qemu-arm -cpu cortex-a15 -L /usr/arm-linux-gnueabihf/"
   # cortexa57 build and fast testsuite (qemu)
   - os: linux
     compiler: aarch64-linux-gnu-gcc
     env: OOT=0 TEST=FAST SDE=0 THR="none" CONF="cortexa57" \
-      PACKAGES="gcc-aarch64-linux-gnu qemu-system-arm qemu-user" \
+      PACKAGES="gcc-aarch64-linux-gnu libc6-dev-arm64-cross qemu-system-arm qemu-user" \
       TESTSUITE_WRAPPER="qemu-aarch64 -L /usr/aarch64-linux-gnu/"
+  # armsve build and fast testsuite (qemu)
+  - os: linux
+    compiler: aarch64-linux-gnu-gcc-10
+    env: OOT=0 TEST=FAST SDE=0 THR="none" CONF="armsve" \
+      PACKAGES="gcc-10-aarch64-linux-gnu libc6-dev-arm64-cross qemu-system-arm qemu-user" \
+      TESTSUITE_WRAPPER="qemu-aarch64 -cpu max,sve=true,sve512=true -L /usr/aarch64-linux-gnu/"
 install:
-- if [ "$TRAVIS_OS_NAME" = "linux" ]; then sudo rm -f /usr/bin/as; fi
-- if [ "$TRAVIS_OS_NAME" = "linux" ]; then sudo ln -s /usr/lib/binutils-2.26/bin/as /usr/bin/as; fi
-- if [ "$TRAVIS_OS_NAME" = "linux" ]; then sudo rm -f /usr/bin/ld; fi
-- if [ "$TRAVIS_OS_NAME" = "linux" ]; then sudo ln -s /usr/lib/binutils-2.26/bin/ld /usr/bin/ld; fi
-- if [ "$CC" = "gcc" ] && [ "$TRAVIS_OS_NAME" = "linux" ]; then export CC="gcc-6"; fi
-- if [ -n "$PACKAGES" ]; then sudo apt-get install -y $PACKAGES; fi
-addons:
-  apt:
-    sources:
-    - ubuntu-toolchain-r-test
-    packages:
-    - gcc-6
-    - binutils-2.26
-    - clang
+- if [ "$CC" = "gcc"  ] && [ "$TRAVIS_OS_NAME" = "linux" ]; then export CC="gcc-8"; fi
+- if [ -n "$PACKAGES" ] && [ "$TRAVIS_OS_NAME" = "linux" ]; then sudo apt-get install -y $PACKAGES; fi
 script:
 - export DIST_PATH=.
-- pwd
 - if [ $OOT -eq 1 ]; then export DIST_PATH=`pwd`; mkdir ../oot; cd ../oot; chmod -R a-w $DIST_PATH; fi
+- $DIST_PATH/configure -t $THR CC=$CC CFLAGS=$CFLAGS $CONF
```

Comment:
Why add `CFLAGS=$CFLAGS`? This doesn't seem to get set to anything special.

Reply:
This is from the old `.travis.yml`.
BLIS' `configure` is not Autoconf so I thought influence of args could be somehow different from env.

If it is not the case, I guess it's enough to just: `$DIST_PATH/configure -t $THR $CONF` ?

Reply:
The old line 74 doesn't have CFLAGS (or at least, what I see here in the diff on the LHS doesn't). I would remove it for consistency.

Reply:
Sorry.
The old `.travis.yml` has only `CC=$CC`.

I used `CFLAGS=-DSVE_XXX` upon other configs but now CFLAGS is not set anyway so this is an unnecessary extra by myself. Will remove.